docs/parameter_files.rst

```diff
@@ -100,18 +100,18 @@ Time
 
     The numerical factors are to convert among seconds, days, and hours. The user must specify sufficient information for the time array to be defined:
     Minimum time requirements:
-        Total time:
-            * (``start_time``, ``end_time``)
-            * (``integration_time``, ``Ntimes``)
-            * (``duration_hours``) or (``duration_days``)
-
-        Time step:
-            * (``duration_hours`` or ``duration_days``, ``Ntimes``)
-            * (``integration_time``)
-
-        Reference time:
-            * (``start_time``)
-            * (``end_time``)
+    Total time:
+        * (``start_time``, ``end_time``)
+        * (``integration_time``, ``Ntimes``)
+        * (``duration_hours``) or (``duration_days``)
+
+    Time step:
+        * (``duration_hours`` or ``duration_days``, ``Ntimes``)
+        * (``integration_time``)
+
+    Reference time:
+        * (``start_time``)
+        * (``end_time``)
```

Comment:
These changes feel like they might mess up the formatting on RTD. Have they been checked there?

Reply:
I just built it on RTD and it's definitely not quite right. Might just need a couple of empty lines added in places.

Reply:
Strangely, what I see on RTD in the `code-format` branch is the same as what I see on the `latest` branch. However, adding in a line break makes the normal statements non-bold, so I've done that.

Reply:
The "total time" line is still appended to the previous line. "Minimum requirements" should maybe also be on it's own line.